api/rs/slint/private_unstable_api.rs

```diff
@@ -213,6 +213,9 @@ pub mod re_exports {
     };
     pub use i_slint_core::slice::Slice;
     pub use i_slint_core::timers::{Timer, TimerMode};
+    pub use i_slint_core::translations::{
+        set_language_internal, translate_bundle, translate_bundle_with_plural,
```

Comment:
The naming really bugs me :). `translate_bundle` literally means that a bundle is translated, but that's not what the function does. It doesn't take a bundle and translates it as a whole. From the given bundle of translations it selects the one that applies to the currently selected language. It could be `select_translation_from_bundle`, `select_translated_string_from_bundle`, or `translate_with_bundle` (to de-emphasise bundle).